src/model-helpers/Document.js

```diff
@@ -0,0 +1,206 @@
+import {
+  getSolidDataset,
+  createThing,
+  buildThing,
+  setThing,
+  getThingAll,
+  createSolidDataset,
+  getStringNoLocale,
+  getUrl,
+  saveSolidDatasetAt,
+  saveFileInContainer,
+  getContainedResourceUrlAll,
+  deleteFile,
+  deleteSolidDataset,
+  getDate
+} from '@inrupt/solid-client';
+
+import sha256 from 'crypto-js/sha256';
+
+import getDriversLicenseData from '../utils/barcode/barcode-scan';
+import formattedDate from '../utils/barcode/barcode-date-parser';
+import { RDF_PREDICATES } from '../constants';
+import { getUserSigningKey, signDocumentTtlFile } from '../utils/cryptography/credentials-helper';
+
+/**
+ * @typedef {import('@inrupt/solid-ui-react').SessionContext} Session
+ */
+
+/**
+ * @typedef {import('../../typedefs').fileObjectType} fileObjectType
+ */
+
+/**
+ * @typedef {import('@inrupt/solid-client').ThingLocal} ThingLocal
+ */
+
+/**
+ * @typedef {import('crypto-js').CryptoJS.lib.WordArray} WordArray
+ */
+
+export const signDocument = async (document, session, containerUrl) => {
+  const signingKey = await getUserSigningKey(session);
+  const signatureDataset = await signDocumentTtlFile(signingKey, document, session, containerUrl);
+  await saveSolidDatasetAt(`${containerUrl}signature.ttl`, signatureDataset, {
+    fetch: session.fetch
+  });
+};
+
+/**
+ * Helper Function that returns Driver's License ttl file based off of image passed
+ *
+ * @function createDriversLicenseTtlFile
+ * @memberof utils
+ * @function createDriversLicenseTtlFile
+ * @param {fileObjectType} fileObject - Object containing information about file
+ * @returns {Promise<ThingLocal>} TTL file Thing - Processes a barcode using zxing
+ * and returns a new TTL file Thing
+ */
+const createDriversLicenseTtlFile = async (fileObject) => {
+  const dlData = await getDriversLicenseData(fileObject.file);
```

Comment:
The function originally takes in the file Blob which were contained in fileObject.

It's now converted into just file, so we can just use `fileObject` for the parameter.

Reply:
fixed